eng/.docsettings.yml

```diff
@@ -121,21 +121,14 @@ known_presence_issues:
 known_content_issues:
   - ['README.md','Root readme']
   - ['doc/README.md','Doc readme']
-  - ['sdk/core/Microsoft.Extensions.Azure/README.md','#5499']
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
-  - ['sdk/eventhub/Azure.Messaging.EventHubs.Shared/README.md','#5499']
```

Comment:
This is the shared code project for Event Hubs, containing only internal types that are used by multiple Event Hubs packages.   It should not be validated as a product ReadMe.

Please consider leaving this exclusion.

Reply:
I have added this back

Reply:
Thanks!